docs/backup-restore.md

```diff
@@ -76,26 +77,27 @@ Please see [the doc](https://github.com/scalar-labs/cassy/blob/master/docs/getti
 ### Cosmos DB
 
 To restore a backup, please follow the [azure official guide](https://docs.microsoft.com/en-us/azure/cosmos-db/restore-account-continuous-backup#restore-account-portal) and change the default consistency to `STRONG` after restoring the backup.
-It is highly recommended to use the midtime of the pause as a restore point as we explained earlier.
```

Comment:
midtime is fine.
I meant, what do you mean by `the midtime of the pause`?
(no midtime in pause since pause is an action)
You meant `the midtime of the paused duration`?

Reply:
>You meant the midtime of the paused duration?

Yes, I will updated it.